### ISSUE TITLE
[Empty State] Add accessibility Sample

### ISSUE THREAD

**Issue by diogofscmariano** [lumada-design/hv-uikit-react#1219]
**Description**
We want a sample in our documentation which shows how can we have accessibility properties added to describe an Empty State. This should be able to happen in the top level, or in either of its elements to enhance its description.

**Acceptance Criteria**
- Add `role=presentation`
- Add sample for Empty which illustrates how to inject accessibility aria attributes and in the action it creates
